packages/@aws-cdk/aws-redshift-alpha/test/cluster.test.ts

```diff
@@ -432,6 +432,39 @@ test('publicly accessible cluster', () => {
   });
 });
 
+test('availability zone relocation enabled', () => {
```

Comment:
nit: For completeness, can we also include testing to make sure `availabilityZoneRelocation` is set to `false` by default?

Reply:
The default value is set to `false`, which depends on CloudFormation specifications.
In CDK, when undefined is passed, it behaves the same as when `false` is set in the deployed cluster.
It's possible to test whether true/false is written in the template when `availabilityZoneRelocation` is set to `true` or `false`.

Also, since this feature doesn't change the cluster's settings itself, it might be difficult to verify if the function is set to `false` in the integration test.
How should we approach this...??

Reply:
Thanks for the explanation. My thought process was having a test to verify that, if we don't provide `availabilityZoneRelocation`, it's set to `false`, but the actual behaviour is that the default false value is being set in the console when it's deployed. Since the other unit/integ tests are passing, we can probably assume that it's being set properly, so a test like this isn't necessary.